UrCan/Blok 2 - Tools/IConvertType.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+
+[InitializeOnLoad]
+public class IConvert {
+	public static Dictionary<string, IConvertType> FILETYPES = new Dictionary<string, IConvertType>();
+	
+	/* THE IMPORTED TYPE CONVERSIONS */
+	/*To add more file type conversions, simply include the implementation of the IConvertType interface and the respective file extension as a string */
+	public static IConvertType[] types = new IConvertType[]{new IJPG(), new IMP3(), new IBMP()};
+
+	static IConvert() {
+		foreach(IConvertType conv in types) FILETYPES.Add(conv.extension.ToLower(), conv);
+	}
+}
+
+[System.Serializable]
+public class Setting {
+	public string name {private set;get;}
```

Comment:
Heb je dit script wel zelf geschreven? Of was het een tutorial?

Reply:
Ja dit heb ik zelf geschreven, Niels :P